Test_misaligned.py

```diff
@@ -0,0 +1,6 @@
+import misaligned
+result, final_string = misaligned.create_printable_string()
+misaligned.print_color_map_on_console(final_string)
+assert(result == 25)
+assert(misaligned.formatter(10, "Black", "Blue").find("|") == misaligned.formatter(0, "White" ,"Blue"))
```

Comment:
Test cases still need to be enhanced to cover boundary cases such as 1,5,6,10,11 etc. The boundary cases are with pair numbers where major color ends and another major color begins.